coffee_core/src/coffee.rs

```diff
@@ -234,6 +234,28 @@ impl PluginManager for CoffeeManager {
         Err(err)
     }
 
+    async fn remove(&mut self, plugin: &str) -> Result<Value, CoffeeError> {
+        debug!("removing plugin: {plugin}");
+        let values: &mut Vec<Plugin> = &mut self.config.plugins;
```

Comment:
```suggestion
        let plugins = &mut self.config.plugins;
```

the compiler will infer the type alone